From 31f890157d7fb701561c92b75960874def041c36 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Thu, 30 Jul 2026 03:20:51 +0200
Subject: [PATCH 13/N] .gitattributes: enforce Unix line endings in .sh
 scripts

Bash is _really_ unhappy with Carriage Returns:

  ../../.././winsup/testsuite/cygrun.sh: 5:
  : not found
  ../../.././winsup/testsuite/cygrun.sh: 8:
  : not found
  ../../.././winsup/testsuite/cygrun.sh: 10:
  : not found
  ../../.././winsup/testsuite/cygrun.sh: 18: Syntax error: end of file unexpected (expecting "then")

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 .gitattributes | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitattributes b/.gitattributes
index 06d51d2..b7e204a 100644
--- a/.gitattributes
+++ b/.gitattributes
@@ -18,3 +18,4 @@
 #     file gets renamed again ...
 
 ChangeLog       merge=merge-changelog
+*.sh eol=lf
